packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -60,13 +60,13 @@ After installing ngrok, ensure that it's running by creating a "tunnel" into a l
 sample-app > ./ngrok http 8082
 ```
 
-or if you installed in your system path:
+Or if you installed in your system path:
 
 ```shell
 sample-app > ngrok http 8082
 ```
 
-or if you used your favorite folder:
+Or if you used your favorite folder:
```

Comment:
Something about favorite folder here seems off.

Could it be something like `Or if you used another directory`?